### PR TITLE
Refs #24121 -- Added __repr__() to StreamingHttpResponse and subclasses.

### PR DESCRIPTION
**Pull request by Nrestrepo05**
Added repr method to the StreamingHttpResponse class

### CODE REVIEW

File: django/http/response.py [hunk 1]
```diff
@@ -402,6 +402,11 @@ def __init__(self, streaming_content=(), *args, **kwargs):
         # See the `streaming_content` property methods.
         self.streaming_content = streaming_content
 
+    def __repr__(self):
+        class_name = self.__class__.__name__
+        status_code = self.status_code
+        return f'<{class_name} status_code={status_code}>'
```
Comment (felixxm) [inline django/http/response.py:408]: I would use the same format as in `HttpResponse.__repr__()`:
```suggestion
        return '<%(cls)s status_code=%(status_code)d%(content_type)s>' % {
            'cls': self.__class__.__qualname__,
            'status_code': self.status_code,
            'content_type': self._content_type_for_repr,
        }
```

File: tests/httpwrappers/tests.py [hunk 2]
```diff
@@ -662,6 +662,11 @@ def test_streaming_response(self):
 
         r = StreamingHttpResponse(iter(['hello', 'world']))
         self.assertEqual(r.getvalue(), b'helloworld')
+    
+    def test_streaming_response_repr(self):
```
Comment (felixxm) [inline tests/httpwrappers/tests.py:666]:
```suggestion
    def test_repr(self):
```

File: tests/httpwrappers/tests.py [hunk 3]
```diff
@@ -662,6 +662,11 @@ def test_streaming_response(self):
...
+        r = StreamingHttpResponse(iter(['hello', 'café']))
+        expected_repr_response = '<StreamingHttpResponse status_code=200>'
+        self.assertEqual(r.__repr__(), expected_repr_response)
```
Comment (felixxm) [inline tests/httpwrappers/tests.py:669]: There is not need for an extra variable (`expected_repr_response`), also, we should call `__repr__` directly:
```suggestion
        self.assertEqual(repr(r), '<StreamingHttpResponse status_code=200>')
```

Comment (felixxm) [reply tests/httpwrappers/tests.py:669]: Can you add a similar test for the `FileResponse`?

**Review by felixxm — COMMENTED** [review]
@Nrestrepo05 Thanks for this patch :+1: Please check my comments.

File: tests/httpwrappers/tests.py (hunk 3 above)
Comment (Nrestrepo05) [reply tests/httpwrappers/tests.py]: sure!

**Comment by Nrestrepo05** [comment]
Already did the changes! Please let me know if there's something else that I need to change. Thanks!!

**Comment by felixxm** [comment]
@Nrestrepo05 Thanks :+1: Welcome aboard :boat: 

I pushed minor edits.

**Comment by Nrestrepo05** [comment]
thanks!!

**Pull request merged**